lib/web/mage/validation.js

```diff
@@ -1877,6 +1878,27 @@
             this._listenFormValidate();
         },
 
+        _workaroundForArrayInputs: function () {
```

Comment:
Please rename the method to something like "prepareArrayInputs" and add the `@see` annotation to doc block with link to issue (https://github.com/jquery-validation/jquery-validation/pull/717)